### PR TITLE
Use Xcode 15.0.1

### PR DESCRIPTION
**Pull request by jkmassel**
### Testing Details

Ensure CI passes

---

- [x] Please check here if your pull request includes additional test coverage.
- [x] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

File: WordPressAuthenticatorTests/Model/LoginFieldsValidationTests.swift [hunk 1]
```diff
@@ -39,7 +39,9 @@ class LoginFieldsValidationTests: XCTestCase {
         loginFields.siteAddress = "https://hostname"
         XCTAssert(loginFields.validateSiteForSignin(), "Since we want to validate simple mistakes, to use a hostname you'll need an http:// or https:// prefix.")
 
-        loginFields.siteAddress = "host name.com"
-        XCTAssertFalse(loginFields.validateSiteForSignin(), "Hostname with spaces should not validate.")
+//      This test is no longer valid on Xcode 15 – `URL` will automatically convert spaces to `%20` in a domain name
+//
+//        loginFields.siteAddress = "host name.com"
+//        XCTAssertFalse(loginFields.validateSiteForSignin(), "Hostname with spaces should not validate.")
```
Comment (spencertransier) [inline WordPressAuthenticatorTests/Model/LoginFieldsValidationTests.swift]: Do we want to leave this commented out or just remove it?

Comment (oguzkocer) [inline WordPressAuthenticatorTests/Model/LoginFieldsValidationTests.swift]:
```suggestion
```

If it's not going to be used, I think we should just delete the code.

File: .buildkite/validate_podspec_annotation.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+**`validate_podspec` was bypassed!**
+
+As of Xcode 14.3, libraries with deployment target below iOS 11 (iOS 12 in Xcode 15) fail to build out of the box.
+The reason is a missing file, `/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc/libarclite_iphonesimulator.a`, more info [here](https://stackoverflow.com/questions/75574268/missing-file-libarclite-iphoneos-a-xcode-14-3).
+
+Client apps can work around this with a post install hook that updates the dependency deployment target, but libraries do not have this option.
+
+Our old Alamofire dependency targets iOS 8.0, making the validation build fail.
+
+In the interest of using up to date CI (i.e. not waste time downloading old images) we bypass validation until either we are able to drop or upgraed Alamofire, or find a workaround.
```
Comment (oguzkocer) [inline .buildkite/validate_podspec_annotation.md]:
```suggestion
In the interest of using up to date CI (i.e. not waste time downloading old images) we bypass validation until either we are able to drop or upgrade Alamofire, or find a workaround.
```

Comment (spencertransier) [inline .buildkite/validate_podspec_annotation.md]:
```suggestion
In the interest of using up to date CI (i.e. not waste time downloading old images) we bypass validation until either we are able to drop or upgrade Alamofire, or find a workaround.
```

**Review by spencertransier — CHANGES_REQUESTED** [review]

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -28,7 +28,12 @@ steps:
   - label: "🔬 Validate Podspec"
     key: "validate"
     command: |
-      validate_podspec
+      # validate_podspec
+      echo '+++ ⚠️ validate_podspec was bypassed ⚠️'
```
Comment (oguzkocer) [inline .buildkite/pipeline.yml:32]: It might be worth adding a link to the `.buildkite/validate_podspec_annotation.md` documentation from here.

**Review by oguzkocer — APPROVED** [review]
I have a couple minor suggestions, but I'll approve to unblock.

**Review by spencertransier — APPROVED** [review]

**Pull request merged**